runtime/src/lib.rs

```diff
@@ -109,7 +109,7 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 /// Max size for serialized extrinsic params for this testing runtime.
 /// This is a quite arbitrary but empirically battle tested value.
 #[cfg(test)]
-pub const CALL_PARAMS_MAX_SIZE: usize = 240;
+pub const CALL_PARAMS_MAX_SIZE: usize = 244;
```

Comment:
nitpick: only change i see in the PR, which doesn't have any corresponding requirement mentioned.